addons/l10n_ec/models/account_chart_template.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+        res = super(AccountChartTemplate, self)._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
+        for journal in res:
+            # TODO figure out why the "account.chart.template" from l10n_ec_edi is being called
+            # TODO move this code to l10n_ec_edi's "account.chart.template" ?
+            if journal.get('code') == 'INV' and company.country_code == 'EC':
```

Comment:
@odoostan  @pepetreshere For the TODO, I tried through installing and what happens is that YourCompany gets installed and has only this journal (which I wonder how as well) and not the others from l10n_ec_edi.  
```suggestion
            if journal.get('code') == 'INV' and company.account_fiscal_country_id.code == 'EC':
```

Reply:
@jco-odoo regarding the suggested change, I will implement it, just first let's clarify the expected behavior when the country is not set but the chart of accounts corresponds to Ecuador, we can review it in tomorrow's call.

Reply:
@jco-odoo it is done, it did worked as expected (the chart of accounts loading methods are called even if the company was not set).